.github/workflows/build-test-release.yml

```diff
@@ -49,47 +49,32 @@ jobs:
   compliance-copyrights:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: apache/skywalking-eyes@v0.4.0
+      - uses: actions/checkout@v4
+      - uses: apache/skywalking-eyes@v0.6.0
   
   pre-commit:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
         with:
           python-version: "3.7"
-      - uses: pre-commit/action@v3.0.0
+      - uses: pre-commit/action@v3.0.1
 
   semgrep:
     runs-on: ubuntu-latest
     name: security-sast-semgrep
-    if: github.actor != 'dependabot[bot]'
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - id: semgrep
         uses: semgrep/semgrep-action@v1
         with:
           publishToken: ${{ secrets.SEMGREP_PUBLISH_TOKEN }}
 
-  review_secrets:
```

Comment:
What is the reason for removing trufflehog job?

Reply:
I mean - this one is old, but imo we should rather replace it with https://github.com/marketplace/actions/trufflehog-oss

Reply:
I don't find it useful and it does not seem to be a requirement to have it.